### PR TITLE
Communication - Added samples for beginCreateSearch

### PR DESCRIPTION
**Pull request by jbeauregardb**
Added beginCreateSearch documentation

### CODE REVIEW

File: sdk/communication/azure-communication-administration/README.md [hunk 1]
```diff
@@ -272,6 +272,29 @@ PhoneNumberClient phoneNumberClient = createPhoneNumberClient();
 phoneNumberClient.configureNumber(phoneNumber, pstnConfiguration);
 ```
 
+## Long Running Operations
+
+The Phone Number Client supports a variety of long running operations that allow indefinite polling time to the functions listed down below. Because these functions may take a long time to resolve, the use of long running operations is encouraged.
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-administration/README.md:277]: encouraged? I thought that is the only way to access these method..

File: sdk/communication/azure-communication-administration/src/samples/java/com/azure/communication/administration/ReadmeSamples.java [hunk 2]
```diff
@@ -141,6 +146,28 @@ public PhoneNumberClient createPhoneNumberClient() {
...
+        HttpClient httpClient = new NettyAsyncHttpClientBuilder().build();
+
+        PhoneNumberAsyncClient phoneNumberClient = new PhoneNumberClientBuilder()
+            .endpoint(endpoint)
+            .accessKey(accessKey)
+            .httpClient(httpClient)
+            .buildAsyncClient();
+
+        return phoneNumberClient;
+    }
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-administration/src/samples/java/com/azure/communication/administration/ReadmeSamples.java:169]: Wondering if really need to have the async client sample here? Maybe just have the sync sample (using syncPoller) and that should be unless you see a value in doing both?

**Review by ankitarorabit — APPROVED** [review]

File: sdk/communication/azure-communication-administration/README.md (hunk 1 above)
Comment (jbeauregardb) [reply sdk/communication/azure-communication-administration/README.md:277]: Didn't know that but makes sense.

Comment (jbeauregardb) [reply sdk/communication/azure-communication-administration/README.md:277]: Would it make sense to then remove the normal createSearch example from the README? @ankitarorabit

Comment (ankitarorabit) [reply sdk/communication/azure-communication-administration/README.md:277]: Yes, I think that should be done.. thanks for pointing out.

**Pull request merged**